keel-sql/src/main/resources/db/changelog/20200511-misc-platform-review-changes.yml

```diff
@@ -0,0 +1,45 @@
+databaseChangeLog:
+- changeSet:
+    id: add-missing-primary-keys
+    author: fletch
+    changes:
+    - dropUniqueConstraint:
+        tableName: artifact_last_checked
+        constraintName: artifact_uid
+        uniqueColumns: artifact_uid
+    - dropUniqueConstraint:
+        tableName: delivery_config_last_checked
+        constraintName: delivery_config_uid
+        uniqueColumns: delivery_config_uid
+    - dropUniqueConstraint:
+        tableName: resource_last_checked
+        constraintName: resource_uid
+        uniqueColumns: resource_uid
+    - addPrimaryKey:
+        tableName: artifact_last_checked
+        columnNames: artifact_uid
+    - addPrimaryKey:
+        tableName: delivery_config_last_checked
+        columnNames: delivery_config_uid
+    - addPrimaryKey:
+        tableName: resource_last_checked
+        columnNames: resource_uid
+    - renameColumn:
+        tableName: event
+        oldColumnName: uid
+        newColumnName: ref
+        columnDataType: varchar(255)
+    - dropIndex:
+        tableName: artifact_versions
+        indexName: artifact_versions_artifact_idx
+    - dropIndex:
+        tableName: delivery_config_artifact
+        indexName: delivery_config_artifact_unique_idx
+    - dropIndex:
+        tableName: environment_resource
+        indexName: environment_resource_unique_idx
```

Comment:
Dropped redundant indices

Reply:
These should be a separate changeset in this changelog, no?

Reply:
P.S. I believe there was a redundant index in `event` as well (`event_scope_uid_idx`).